stable/kube-state-metrics/README.md

```diff
@@ -55,6 +55,8 @@ $ helm install stable/kube-state-metrics
 | `collectors.secrets`                  | Enable the secrets collector.                           | true                                        |
 | `collectors.services`                 | Enable the services collector.                          | true                                        |
 | `collectors.statefulsets`             | Enable the statefulsets collector.                      | true                                        |
+| `collectors.ingresses`                | Enable the ingresses collector.                         | true                                        |
```

Comment:
we should follow the alphabetical order here.

Reply:
done